pandas/tests/indexes/timedeltas/test_arithmetic.py

```diff
@@ -792,7 +792,7 @@ def test_addition_ops(self):
         pytest.raises(ValueError, lambda: tdi[0:1] + dti)
 
         # random indexes
-        pytest.raises(TypeError, lambda: tdi + Int64Index([1, 2, 3]))
```

Comment:
is this an api change?

Reply:
not sure if it counts as api change or bugfix.  The error currently being raises is incorrect.